package.json

```
@@ -46,6 +46,7 @@
     "index.js"
   ],
   "dependencies": {
+    "@types/unist": "^2.0.5",
```

keep the semver range wide to prevent duplicate versions, where possible 
```suggestion
    "@types/unist": "^2.0.0",
```